src/SpineOpt.jl

```diff
@@ -128,7 +129,13 @@ include("constraints/constraint_node_state_capacity.jl")
 include("constraints/constraint_ratio_unit_flow.jl")
 include("constraints/constraint_ratio_out_in_connection_flow.jl")
 include("constraints/constraint_connection_flow_capacity.jl")
-include("constraints/constraint_connection_flow_ptdf.jl")
+include("constraints/constraint_connection_flow_capacity_strict.jl")
+include("constraints/constraint_connection_intact_flow_capacity.jl")
+include("constraints/constraint_connection_flow_intact_flow.jl")
```

Comment:
Maybe we can rename this to `constraint_connection_intact_flow` to avoid repetition of the word flow?

Reply:
I was trying to capture that the constraint is capturing the relationship between the two flows - perhaps `constraint_connection_flow_to_intact_flow` does that read better?